components/admin_console/database_settings.jsx

```diff
@@ -280,13 +282,12 @@ export default class DatabaseSettings extends AdminSettings {
                             defaultMessage='Minimum number of characters in a hashtag. This must be greater than or equal to 2. MySQL databases must be configured to support searching strings shorter than three characters, <link>see documentation</link>.'
                             values={{
                                 link: (msg) => (
-                                    <a
+                                    <ExternalLink
+                                        location='database_settings'
```

Comment:
Nothing to change here, but seeing these `location` things repeated does make me wonder if we'd benefit from wrapping the entire contents of these screens/modals/components in a Context that provides this. That'd be overkill to just save us from having to pass the same location repeatedly for multiple `ExternalLink`s, but I could see that sort of pattern being useful for other things in the future

Reply:
Something like a "breadcrumb" in a context would be pretty cool for sure. I know there are some patterns around Cloud telemetry that sort of mimics that by appending `'location1 > location2' + 'location3'` through props, which is a bit messy imo